src/components/RewardsCard/index.module.scss

```diff
@@ -0,0 +1,49 @@
+.cardBase {
+  width: 271px;
+  // should be 379px when we update it to use two stickers
+  height: 339px;
```

Comment:
can we not hardcode the height and have it fit the content?

Reply:
Hmm, was my first try but since we don't exactly know the size of the images that will come, and in the case mentioned above, the cards won't follow the same size.

Reply:
![image](https://github.com/user-attachments/assets/a454f727-142c-40f0-9198-682480fd1f36)